styles/lobby.module.scss

```diff
@@ -1,5 +1,6 @@
 @use 'variables.module' as *;
 @use 'mixins.module' as *;
+@use 'LobbyCircle.module' as *;
```

Comment:
oh damn.. okay i dont know how i didn't notice this..

preferably only the variables.modules file will be *.. everything else should follow modular CSS..

```ts
@use 'mixins.module' as mixin;
``` 

and I think we can put LobbyCircle variables into the variable CSS file!